app/views/posts/index.html.erb

```diff
@@ -19,8 +19,13 @@
         <h5 class="card-title"><%= post.title%></h5>
```

Comment:
👍 Good work so far! However, there's a small redirection issue that needs attention.

- [ ] [Optional] Currently, from the `posts#index` page, when clicking on a post, it redirects to the `users#show` page. As per the project requirement, it should redirect to the `posts#show` page. Please review your code to identify and fix this redirection error.

https://github.com/Surafels/Blog-app/assets/55840999/8e39e7e0-e4c9-4cdd-ad89-5ad1fe1d3c0f

Keep up the progress! If you have any questions or need further assistance, feel free to ask. 🚀👍